licenses/restrictiveness.go

```diff
@@ -0,0 +1,55 @@
+package licenses
+
+type LicenseRestrictiveness string
+
+const (
+	RestrictionsNone         LicenseRestrictiveness = "None"
+	RestrictionsShareLicense LicenseRestrictiveness = "ShareLicense"
+	RestrictionsShareCode    LicenseRestrictiveness = "ShareCode"
+	RestrictionsUnknown      LicenseRestrictiveness = "Unknown"
+	RestrictionsNotAllowed   LicenseRestrictiveness = "NotAllowed"
+)
+
+func LicenseTypeRestrictiveness(licenseTypes ...Type) LicenseRestrictiveness {
+	if len(licenseTypes) == 0 {
+		return RestrictionsNone
```

Comment:
When there are no licenses, we are not allowed to use the code, because we don't know. We should return Unknown in this case.

Reply:
@Bobgy Fixed in https://github.com/google/go-licenses/pull/203/commits/f528a3fdd8211e77dc2c68883a94c6ce4bf8cecf